docs/components/card/index.html

```diff
@@ -0,0 +1,142 @@
+          corporis architecto saepe corrupti sunt illum fuga aliquam error
+          eveniet iste, maiores quaerat? Molestiae, porro? Esse iure sint.
+        </p>
+        <button class="button button-link button-small">Read More >></button>
+      </div>
+    </div>
+  </div>
+
+  <iframe
+    src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C0%29&t=a11y-dark&wt=none&l=htmlmixed&width=888.75&ds=true&dsyoff=20px&dsblur=79px&wc=true&wa=false&pv=34px&ph=52px&ln=true&fl=1&fm=Hack&fs=15.5px&lh=163%25&si=false&es=2x&wm=false&code=%253C%21---%2520Basic%2520Vertical%2520Text%2520Only%2520Card%2520---%253E%250A%253Cdiv%2520class%253D%2522card%2520card-vertical%2520notched-corner%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522card-header%2522%253E%250A%2520%2520%2520%2520%253Ch3%253ECard%2520Title%253C%252Fh3%253E%250A%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522card-content%2522%253E%250A%2520%2520%2520%2520%253Cp%253E%250A%2520%2520%2520%2520%2520%2520%257B........%257D%250A%2520%2520%2520%2520%253C%252Fp%253E%250A%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522card-footer%2522%253E%250A%2520%2520%2520%2520%253Cbutton%2520class%253D%2522button%2520button-link%2520button-small%2522%253ERead%2520More%2520%253E%253E%253C%252Fbutton%253E%250A%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A%250A%253C%21---%2520Basic%2520Horizontal%2520Card%2520with%2520dismiss%2520Button%2520--%253E%250A%253Cdiv%2520class%253D%2522card%2520card-horizontal%2520border-radius-sm%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522card-dismiss-button%2522%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522fa-regular%2520fa-circle-xmark%2520fa-xl%2522%253E%253C%252Fi%253E%250A%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522card-header%2522%253E%250A%2520%2520%2520%2520%253Cimg%2520src%253D%2522https%253A%252F%252Fpicsum.photos%252F200%2522%2520alt%253D%2522%2522%2520class%253D%2522border-radius-sm%2522%2520%252F%253E%250A%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522card-content%2522%253E%250A%2520%2520%2520%2520%253Cp%253E%250A%2520%2520%2520%2520%2520%257B.......%257D%250A%2520%2520%2520%2520%253C%252Fp%253E%250A%2520%2520%2520%2520%253Cbutton%2520class%253D%2522button%2520button-link%2520button-small%2522%253ERead%2520More%2520%253E%253E%253C%252Fbutton%253E%250A%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+    class="code-snippets code-snippets-md"
+    sandbox="allow-scripts allow-same-origin"
+  >
+  </iframe>
+
+  <h2 class="margin-vertical-3 margin-top-5">Card with Image & Badge</h2>
+  <p>These types of card can be used for product details.</p>
+  <div class="cards-group margin-vertical-4 border-radius-sm cards-group-2">
+    <div class="card card-vertical border-radius-sm card-image card-shadow">
+      <div class="card-header">
+        <img
+          src="https://picsum.photos/seed/picsum/150/150"
+          alt=""
```

Comment:
please put some data in alt tag do not leave it empty

Reply:
Okay Bro Will keep that in mind.